tests/integration/device_test.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+include './tests/unit/utils.sh'
+include './tests/integration/utils.sh'
+include './src/device_info.sh'
+
+function oneTimeSetUp()
+{
+  setup_container_environment
+}
+
+# test if `kw device --local in the container matches the host
+function device_info_test_helper()
+{
+  local distro="${1}"
+  local container
+  local buffer
+  local filter
+  local filter_regex
+  local output
+  local expected_output
+
+  # buffer with all information
+  container="kw-${distro}"
+  buffer=$(container_exec "$container" kw device --local)
+
+  # some fields must be ignored because they surely won't match
```

Comment:
To solve this problem, you can just get that info directly from the container.

Reply:
I think I did not understand  what you mean.

If I manually get the hardware information from the container, won't that be code duplication of the device module? I'd be running the same code of device module, but directly in the container. The purpose shouldn't be test that the output of `device --local` in the container matches the hardware information of the host?